src/chessground/drag.ts

```diff
@@ -178,7 +178,7 @@ export function end(ctrl: Chessground, e: TouchEvent) {
   // board editor mode: delete any piece dropped off the board
   else if (cur.started && draggable.deleteOnDropOff) {
     state.pieces.delete(cur.orig)
-    setTimeout(state.events.change || util.noop, 0)
+    setTimeout(() => (state.events.change || util.noop)(), 0)
```

Comment:
why this change?

Reply:
https://eslint.org/docs/rules/no-implied-eval

Reply:
but I don't get it, as here this is not a string argument that is passed to `setTimeout`, it is a function. I'm not a fan of this change really.

Reply:
I think you're right, it's a bug in the rule... the type of the expression before this change is `() => void`, and you can "fix" the error by assigning the expression to a const first
```typescript
const fn = state.events.change || util.noop
setTimeout(fn, 0)
```

Reply:
then I much prefer to have this instead of the arrow function wrapper

Reply:
https://github.com/typescript-eslint/typescript-eslint/issues/3621

Reply:
Just backed out this change, we can re-enable the rule later once the bug is fixed